BUILDING.md

```diff
@@ -223,18 +223,17 @@ Consult previous versions of this document for older versions of Node.js:
 
 ### Note about Python 2 and Python 3
 
-The Node.js project uses Python as part of its build process and has
+The Node.js project uses Python as part of its build process and was
 historically only been Python 2 compatible.
```

Comment:
```suggestion
historically only Python 2 compatible.
```

Reply:
you could also just delete the historical note, and say how it is now:
> The Node.js project uses Python as part of its build process.

Reply:
Yeah, I'd second Sam's comment. There's little benefit (and some risk of confusion) in mentioning how things were. Just say how they are.

Reply:
(My comment is not blocking, just in case Michael or anyone else thinks it better the current way.)